pkg/kubelet/apis/config/types.go

```diff
@@ -195,6 +195,10 @@ type KubeletConfiguration struct {
 	// imageMinimumGCAge is the minimum age for an unused image before it is
 	// garbage collected.
 	ImageMinimumGCAge metav1.Duration
+	// ImageMaximumGCAge is the maximum age an image can be unused before it is garbage collected.
+	// The default of this field is 0, which disables it.
+	// +optional
```

Comment:
Not sure if this is WIP but I would expect to see someplace where we default here.

Do you need to add some code in apis/config/v1beta1 to default this to 0?